### PR TITLE
[DRAFT] [onert] Introduce trix loader

### PR DESCRIPTION
**Pull request by glistening**
It introduces trix loaders. Not finished yet.

Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

### CODE REVIEW

**Comment by chunseoklee** [comment]
@glistening PR is exactly the same as the patch in mind. Thank you.

**Comment by glistening** [comment]
Fixed android build ( caused by unused member warning ).

**Comment by glistening** [comment]
@nnfw-bot test cross-runtime-test-debug

File: runtime/onert/frontend/trix/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+set(SOURCES src/trix_loader_base.cc)
+
+nnfw_find_package(TRIXEngine EXACT 2.5.0 QUIET)
+if(TRIXEngine_FOUND)
+  message(WARNING "TRIXEngine found")
+  list(APPEND SOURCES src/trix_loader.cc)
+else()
+  message(WARNING "TRIXEngine NOT found, trix loader is dummy.")
+  list(APPEND SOURCES src/trix_loader_dummy.cc)
```
Comment (glistening) [inline runtime/onert/frontend/trix/CMakeLists.txt:13]: If TrixEngine is not available, dummy trix loader will be used. It has no dependency to trix_engine. Literally, it is a dummy. : )

File: runtime/onert/frontend/trix/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+endif ()
+
+set(SOURCES src/trix_loader_base.cc)
+
+nnfw_find_package(TRIXEngine EXACT 2.5.0 QUIET)
+if(TRIXEngine_FOUND)
+  message(WARNING "TRIXEngine found")
+  list(APPEND SOURCES src/trix_loader.cc)
+else()
+  message(WARNING "TRIXEngine NOT found, trix loader is dummy.")
```
Comment (glistening) [inline runtime/onert/frontend/trix/CMakeLists.txt:12]: I will remove this and "TRIXEngine found" message. It is for me to confirm it works well.

File: runtime/onert/frontend/trix/src/trix_loader.cc [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+private:
+  const char *_model_path;
+  npubin_meta *_meta = nullptr;
+};
+
+void TrixMetaReader::open()
+{
+  assert(_model_path);
+  auto _meta = getNPUmodel_metadata(_model_path, false);
```
Comment (jyoungyun) [inline runtime/onert/frontend/trix/src/trix_loader.cc:54]:
```suggestion
  _meta = getNPUmodel_metadata(_model_path, false);
```

**Comment by glistening** [comment]
Done

**Pull request closed without merging**